### DISCUSSION TITLE
WYSIWYG

### DISCUSSION THREAD

**Discussion by SerhatReis** [twentyhq/twenty#6888] [Ideas]
Is it possible to add a WYSIWYG editor as a fieldtype to an object? An editor similar to the Notes editor.

**Comment by BOHEUS** [comment]
Now it's possible #18634 
<img width="1680" height="1014" alt="image" src="https://github.com/user-attachments/assets/90f61f24-3dbd-46f6-86d7-10d40074051e" />
